cmd/integration/commands/stages.go

```diff
@@ -908,7 +913,11 @@ func stagePolygonSync(db kv.RwDB, ctx context.Context, logger log.Logger) error
 		}
 
 		stageState := stage(stageSync, tx, nil, stages.PolygonSync)
-		cfg := stagedsync.NewPolygonSyncStageCfg(logger, chainConfig, nil, heimdallClient, nil, 0, nil, blockReader, nil, 0, unwindTypes)
+		heimdallStore := heimdall.NewDbStore(db)
+		bridgeStore := bridge.NewDbStore(db)
```

Comment:
I suspect these 2 heimdall and bridge stores here need to be wrapped with a NewSnapshotStore?